### ISSUE TITLE
[Design Deliverable] Add FAQ & Release Notes to the knowledge center

### ISSUE THREAD

**Issue by reitermb** [raft-tech/TANF-app#2371]
**Description:**

Follow-on from #2157. Delivers new FAQ page to the knowledge center. MVP. Don't spend time on flashiness 

**AC:**

- [x] The design is usable and accessible, meaning it adheres to definition of done standards for design work.
> <details>
>      <summary>Expand Definition of Done Standards</summary>
> 
> - It uses [USWDS components and follows it’s UX guidance](https://designsystem.digital.gov/components/), or a deviation is clearly documented
> 
> - Language is intentional and [plain](https://plainlanguage.gov/guidelines/); placeholders are clearly documented
> - It follows [accessibility guidelines](https://accessibility.digital.gov/) and accessibility implementation notes are documented (e.g. clear information hierarchy, color is not the only way meaning is communicated, etc.)
> - If feedback identifies bigger questions or unknowns, create additional issues to investigate
> </details>
- [x] Raft Review has occurred to ensure DoD Standards

**Tasks:**

- [x] Draft a hack.md of FAQ content https://hackmd.io/bhWofQV9Qz-dYPXt7SEG3w
- [x] Draft changes to test environment
- [x] Draft PR to raft-tech 
- [x] jquery version bump
- [x] 3.0 Changelog

**Supporting Documentation:**
- [Hack.md draft](https://hackmd.io/@mreiter/Skg_rKTsi/edit)

**Comment by reitermb** [comment]
@ADPennington @lfrohlich ready for review

**Comment by reitermb** [comment]
@ADPennington @lfrohlich Following up, the FAQ page is now live in the test environment as well as the hack.md. https://reitermb.github.io/Test/knowledge-center/faq.html

**Comment by lfrohlich** [comment]
Added edits/comments in hack.md. may take another pass Monday

**Comment by ADPennington** [comment]
thank you. i left feedback in the hackmd @reitermb

**Comment by reitermb** [comment]
All notes have been responded to & addressed in revisions. The hack.md and the Test environment are both live with the changes. 

Synced with Alex after Office Hours Friday and the main outstanding change was around how to reflect the complexity of FAQ #1. Not burying the lead that most grantees don't need to change anything but also speaking to the fact that we require changes from the subset of folks resubmitting partial data. We discussed splitting the FAQ into two with one speaking to first submission for a quarter and one speaking to resubmissions but I think I've found a solid middle ground that speaks to both in the one question without being too wordy. 

Thoughts welcomed! 

cc @lfrohlich @ADPennington

**Comment by lfrohlich** [comment]
Added a few suggested edits to the first question, "How much data can I submit per day?" and "When can I expect to receive feedback on my submitted data?" and the last release note. Eventually it may be worth numbering or grouping the FAQS, but not necessary now. With these additional changes and after a11y review, should be good to go @reitermb

**Comment by ttran-hub** [comment]
Thanks @reitermb @lilybl1, Updates to the knowledge center are A11y ready. cc @lfrohlich @ADPennington

**Comment by lfrohlich** [comment]
This was merged. Thanks!
